source/manual/get-ssh-access.html.md

```diff
@@ -74,4 +74,11 @@ You should be able to do the same thing in your VM:
 
 The built-in key-forwarding should mean that you don't need to edit the `~/.ssh/config` file inside the VM (it will default to your host machine's config file instead).
 
+## Running a console
+Once you have SSH'd into a machine, you can also open a console for a particular application so you can execute commands, for example:
+
+```
+govuk_app_console publishing-api
```

Comment:
We already have this [in the SSH to AWS](https://docs.publishing.service.gov.uk/manual/howto-ssh-to-machines-in-aws.html#application-node) docs. 🤔

Reply:
That's true, although it's not AWS specific?

Reply:
It's really not AWS specific. Strange why it's there. I'm up for putting it here in the place where people most often learn about SSH. :+1: